src/services/sanity.js

```diff
@@ -1301,6 +1310,42 @@ export async function fetchSanity(query, isList) {
   }
 }
 
+async function needsAccessDecorator(results)
+{
+    if (globalConfig.sanityConfig.useDummyRailContentMethods) return results;
```

Comment:
_Unresolved variable useDummyRailContentMethods_ - I assume this just needs to be added to the doc block in config.js?